src/main/java/org/folio/services/budget/BudgetExpenseClassService.java

```diff
@@ -110,11 +117,14 @@ private Future<Void> deleteBudgetExpenseClasses(List<BudgetExpenseClass> deleteL
   }
 
   private Future<Void> checkNoTransactionsAssigned(List<BudgetExpenseClass> deleteList, SharedBudget budget, RequestContext requestContext) {
+    log.debug("checkNoTransactionsAssigned:: Checking there is no transaction assigned for budget: {} and deleteList with size: {}", budget.getId(), deleteList.size());
     return transactionService.retrieveTransactions(deleteList, budget, requestContext)
       .map(transactions -> {
         if (isNotEmpty(transactions)) {
+          log.error("checkNoTransactionsAssigned:: There is assigned transaction");
```

Comment:
debug level will not be enabled by default, and we have budgetId only in the first debug log, so better to add budgetId to this error log

Reply:
thanks, added